go.mod

```diff
@@ -183,6 +183,7 @@ require (
 	github.com/Microsoft/go-winio v0.5.1 // indirect
 	github.com/antihax/optional v1.0.0 // indirect
 	github.com/gofrs/uuid v3.2.0+incompatible // indirect
+	github.com/gofrs/uuid/v5 v5.0.0
```

Comment:
Seems weird to have 2 different versions of the same uuid package. Even if one is indirect

Reply:
@cristiGuranIonos indeed, I used the indirect one and the generation is done correctly.